src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const { id, title, completed, userId } = todo;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(userId)
```

Comment:
It is a good practice to handle errors when making API calls. You should add a .catch() block to handle any errors that may occur when fetching the user.